README.md

```diff
@@ -132,11 +133,25 @@ The `fugue` API consists of four functions:
 -   `fugue.missing_resource(resource_type)` marks a resource as **missing**.
     This is useful if you for example _require_ a log group to be present.
 
-## Control mapping
+### Rule library
+
+| Provider | Service    | Rule Name                              | Rule Summary                                                                                               |
+|----------|------------|----------------------------------------|------------------------------------------------------------------------------------------------------------|
+| AWS      | IAM        | iam\_user\_attached_policy              | IAM policies should not be attached directly to users                                                      |
+| AWS      | IAM        | iam\_admin\_policy                       | IAM policies should not have full "*:*" administrative privileges                                          |
+| AWS      | VPC        | vpc\_flow\_logging_enabled               | VPC flow logging should be enabled                                                                         |
+| AWS      | VPC        | security\_group\_no\_ingress_22           | VPC security group rules should not permit ingress from '0.0.0.0/0' to port 22 (SSH)                       |
+| AWS      | VPC        | security\_group\_no\_ingress\_3389         | VPC security group rules should not permit ingress from '0.0.0.0/0' to port 3389 (Remote Desktop Protocol) |
+| AWS      | VPC        | security\_group\_ingress\_only\_80\_443     | VPC security group rules should not permit ingress from '0.0.0.0/0' except to ports 80 and 443             |
+| AWS      | CloudTrail | cloudtrail\_log\_file\_validation\_enabled | CloudTrail log file validation should be enabled                                                           |
+| AWS      | KMS        | kms\_rotate                             | KMS CMK rotation should be enabled                                                                         |
+| AWS      | EBS        | ebs\_volume\_encrypted                   | EBS volume encryption should be enabled
+
+## Compliance controls
 
 ```ruby
 In Regula, _rules_ provide the lower-level implementation details, and
-_controls_ are policies that map to sets of rules.  Controls can
+_controls_ are compliance controls (e.g. CIS AWS Foundations Framework 4-1) that map to sets of rules.  Controls can
```

Comment:
Add comma after `e.g.` and change name: `e.g., CIS AWS Foundations Benchmark 4-1`